ros_ws/src/simulation/simulation_tools/src/simulation_tools/log_stats.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python
+
+import rospy
+import os
+import sys
+from datetime import datetime
+from rospkg import RosPack
+from simulation_tools.track import track
+from simulation_tools.reset_car import Point
+from drive_msgs.msg import drive_param,gazebo_state_telemetry
+from gazebo_msgs.msg import ModelStates
+from jsk_rviz_plugins.msg import OverlayText
+from jsk_rviz_plugins.overlay_text_interface import OverlayTextInterface
+
+TOPIC_DRIVE_PARAMETERS = "/input/drive_param/autonomous"
+TOPIC_GAZEBO_MODEL_STATE = "/gazebo/model_states"
+TOPIC_GAZEBO_STATE_TELEMETRY = "/gazebo/state_telemetry"
+
+def __init__(self):
```

Comment:
Wozu ist das?

Reply:
Ab Zeile 262 gibt es schon eine Initialisierung der Variablen

Reply:
das ist teilweise von dir - das file muss so, da er sonst viel zu häufig über nicht initialisierte variablen fällt, daher initialisiere ich das beim laden schon mal - verstehe das auch nicht, aber dann ist die wahrscheinlichkeit gering wenn ich das rausnehme, dass es läuft ...

Reply:
Aber irgendwas stimmt doch hier nicht, oder? Müssen die denn global sein?

Reply:
Hier in init ja. Ab Zeile 261 eigentlich nicht, weil die schon automatisch global sind. Ein init ohne umgebende Klasse hab ich auch noch nicht gesehen und ich hab auch nichts dazu gefunden. Ich vermute mal, dass die hier nicht aufgerufen wird. Zumindest passiert das in meinen Testbeispiel nicht.